ca/django_ca/utils.py

```diff
@@ -630,8 +633,26 @@ def write_private_file(path, data):
     """Function to write binary data to a file that will only be readable to the user."""
 
     try:
-        with os.fdopen(os.open(path, os.O_CREAT | os.O_WRONLY, 0o400), 'wb') as fh:
-            fh.write(data)
+        if isinstance(path, FieldFile):
```

Comment:
If we are storing from the model, it will be a FieldFile, but if we are storing from somewhere else, like a command, it can be a path (relative in the default_storage). This check is because of that.